src/server/chromium/crPage.ts

```diff
@@ -764,7 +764,12 @@ class FrameSession {
       this._screencastState = 'started';
       this._videoRecorder = videoRecorder;
       this._screencastId = screencastId;
-      this._crPage._browserContext._browser._screencastStarted(screencastId, options.outputFile, this._page);
+      const screencast = this._crPage._browserContext._browser._screencastStarted(screencastId, options.outputFile);
+      this._crPage.pageOrError().then(pageOrError => {
+        //     await new Promise(makeWaitForNextTask());
```

Comment:
hacking the event order? it might not work.

Reply:
This has been reverted.